code/modules/reagents/Chemistry-Reagents/Chemistry-Reagents-Medicine.dm

```diff
@@ -278,6 +278,7 @@
 
 /datum/reagent/tramadol/morphine
 	name = "Morphine"
+	reagent_addiction_strength = 15
```

Comment:
While realistic, everyone takes already morphine in this game a lot. People are already basically addicted to it. I would leave addiction for mood boosters like cigarettes and booze, not drugs that are required to function.